schemas/telemetry/configuration-schema.json

```diff
@@ -49,6 +49,12 @@
                   "minimum": 0,
                   "maximum": 100
                 },
+                "trace_context_injection": {
+                  "type": "integer",
+                  "description": "The opt-in configuration to add trace context",
+                  "minimum": 0,
+                  "maximum": 1
+                },
```

Comment:
**💬 suggestion:** ‏following https://github.com/DataDog/browser-sdk/pull/2656#discussion_r1531774092, `trace_context_injection` should be a `string`, not a `number`.